quickstart/101-devtest-labs/TestRecord.md

```diff
@@ -0,0 +1 @@
+Article not yet tested by automated testing pipeline.
```

Comment:
You don't need this empty TestRecord.md file, the pipeline would generate one once this pr got merged.

Reply:
Fixed.